src/components/person/PersonInfo.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Grid, Theme, Typography } from '@mui/material'
+import createStyles from '@mui/styles/createStyles'
+import makeStyles from '@mui/styles/makeStyles'
+import theme from 'common/theme'
+import { format, parseISO } from 'date-fns'
+import { PersonResponse } from 'gql/person'
+import { useTranslation } from 'next-i18next'
+import React from 'react'
+
+type Props = {
+  person: PersonResponse
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    infoHeading: {
+      marginBottom: theme.spacing(2),
+      marginTop: theme.spacing(2),
+    },
+    infoWrapper: {
+      '&>*': {
+        marginBottom: theme.spacing(1),
+      },
+    },
+  }),
+)
+
+function PersonInfo({ person }: Props) {
+  const classes = useStyles()
+  const { t } = useTranslation()
+  return (
+    <Grid container>
+      <Grid item xs={12} md={6}>
+        <Typography className={classes.infoHeading} variant="h6" color={theme.palette.primary.dark}>
+          {t('person:info.contact')}
+        </Typography>
+        <Box className={classes.infoWrapper}>
+          <Typography>
+            {t('person:info.email')}: {person.email}
+          </Typography>
+          <Typography>
+            {t('person:info.tel')}: {person.phone}
+          </Typography>
+          <Typography>
+            {t('person:info.address')}: {person.address}
+          </Typography>
+        </Box>
+      </Grid>
+      <Grid item xs={12} md={6}>
+        <Typography className={classes.infoHeading} variant="h6" color={theme.palette.primary.dark}>
+          {t('person:info.general')}
+        </Typography>
+        <Box className={classes.infoWrapper}>
+          <Typography>
+            {t('person:info.createdAt')}: {format(parseISO(person.createdAt), 'PPpp')}
```

Comment:
There are couple of pre-existing formatting functions at https://github.com/podkrepi-bg/frontend/blob/master/src/common/util/date.ts

Let's reuse one of them if matches the expected format or add this format to the utils so it can be reused later on.

Reply:
Done in [95fedb9](https://github.com/podkrepi-bg/frontend/pull/728/commits/95fedb9d7a1c818ab2b7b06e662d78ad23679d4e)